lib/internal/net.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
@jasnell While it is small enough now for that, I have been working on cleaning up some of the logic in net and moving some more into internal. Either way works for me though, so if this is something you feel strongly about, I'll be happy to change.